src/features/movies/MoviesList/index.js

```diff
@@ -61,11 +57,14 @@ const MoviesList = ({ insideDetails }) => {
               <StyledLink key={movie.id} to={toMovieDetails({ id: movie.id })}>
                 <TileMovie>
                   <ImageWrapper>
-                    <Image src={`https://image.tmdb.org/t/p/w400${movie.poster_path}`} alt="" />
+                    {movie.poster_path ?
+                      <Image src={`https://image.tmdb.org/t/p/w400${movie.poster_path}`} /> :
```

Comment:
In my opinion we should have alt="" in Image but it is your decision according to the call.

Reply:
I restored alt with "Movie poster" description.